packages/grid-pro/theme/lumo/vaadin-grid-pro-edit-select.js

```diff
@@ -1,5 +1,3 @@
-import '@vaadin/item/theme/lumo/vaadin-item.js';
-import '@vaadin/list-box/theme/lumo/vaadin-list-box.js';
```

Comment:
This could be extracted into separate PR. Not a blocker though.

Reply:
Yeah, this change should have been part of https://github.com/vaadin/web-components/pull/6747

Reply:
Added the PR here https://github.com/vaadin/web-components/pull/6763 (also did this change for the material theme version)